File: ShipmentSrvc/src/main/java/smpl/shipment/Utility.java
package smpl.shipment;

public class Utility
{
	
    public static int validateStringField(String field, String fieldName, int count, StringBuilder errors)
    {
    	int countErr=count;
        if (isNullOrEmpty(field))
        {
            if (countErr == 0)
            {
                errors.append(String.format("\"Empty %s field\"", fieldName));
            }
            else
            {
                errors.append(String.format(",\"Empty %s field\"", fieldName));
            }
            countErr += 1;
        }
        return countErr;
    }

    public static boolean isNullOrEmpty(String str)
    {
        return str == null || str.isEmpty();
    }
}

File: DealerService/Models/Utility.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Text;

namespace DealerApi.Models
{
    public class Utility
    {
        public static int validateStringField(String field, String fieldName, int count, StringBuilder errors)
        {
            if (isNullOrEmpty(field))
            {
                if (count == 0)
                {
                    errors.Append(String.Format("\"Empty %s field\"", fieldName));
                }
                else
                {
                    errors.Append(String.Format(",\"Empty %s field\"", fieldName));
                }
                count += 1;
            }
            return count;
        }

        public static Boolean isNullOrEmpty(String str)
        {
            return str == null || String.IsNullOrEmpty(str);
        }


    }
}
